Formula/tag.rb

```diff
@@ -17,6 +17,7 @@ class Tag < Formula
   def install
     system "make"
     bin.install "bin/tag"
+    man1.install "Tag/tag.1"
```

Comment:
Let's try to figure out how to use `make install` instead which will avoid future omissions: https://github.com/jdberry/tag/blob/master/Makefile#L36